### ISSUE TITLE
understanding of axi_lsu_dma_bridge 

### ISSUE THREAD

**Issue by vignajeth** [chipsalliance/Cores-SweRV-EH2#31]
Hi, 
   I notice that the module `axi_lsu_dma_bridge`  in testbench has one master (lsu) and two slaves ( s0 - main memory, s1 - dma).
   When I looked into the code I observed that when the master provided address falls into iccm address range,  the bridge directs the request to dma slave.

right now the TB directly dumps the data into dccm, iccm. but in real case scenario with the use of lw, sw instructions pointing to dccm address it is possible to write/read data from dccm.

However, to write data into iccm, the data should be moved through lsu and is written in iccm through dma using this bridge right ?
in case of read to iccm, the lsu read request is sent to dma slave , the dma slave will fetch data from iccm and give it to lsu right ?

In short, can i say that this bridge is used for reading/writing iccm ?

let me know if my understanding is write / wrong 😃

**Comment by aprnath** [comment]
Yes, your understanding is correct. This bridge is an external splitter component that helps provide a loopback path to do load/stores to the ICCM via the DMA port. The ICCM would otherwise not be accessible other than via debugger or needing an external DMA master in the SOC.

**Comment by vignajeth** [comment]
Thanks for your reply
I also noticed that this question was asked before #17
